polaris-tokens/tests/test.test.js

```diff
@@ -0,0 +1,5 @@
+describe('foo', () => {
+  it('passes', () => {
+    expect(1).toBe(1);
+  });
+});
```

Comment:
This could be scrappy if we wanted... e.g. input of this JS gives me string output. Have a look at the tests in `stylelint-polaris` we did something similar.